### ISSUE TITLE
Clarify what "fast" endpoint is

### ISSUE THREAD

**Issue by acanidio-econ** [cowprotocol/docs#364]
https://github.com/cowprotocol/docs/blob/01b0d1b6ad60eabd35173a3c7f7519f66df990cc/docs/mevblocker/concepts/endpoints.md?plain=1#L11

Someone should confirm this, but I guess we do not send all txs to searchers if they are send via the "fast" endpoint. Otherwise it is unclear what the difference is with the "max refund" endpoint

**Comment by harisang** [comment]
Maybe @fleupold can clarify this? Also see #365

**Comment by fleupold** [comment]
Yes, for fast endpoint we use heuristics to decide whether transactions are likely going to receive backruns and/or could be attempted to be frontrun out of protocol. Based on the risk/reward we may not share the transaction with searchers. For max refund we always share with searchers.

On Friday, we also discussed potentially delaying max_refund txs if they are sent too late in the block (to not give builders/searchers an excuse to backrun internally), but this hasn't been implemented yet.
